static/app/views/replays/detail/header/replayItemDropdown.tsx

```diff
@@ -0,0 +1,127 @@
+import styled from '@emotion/styled';
+
+import {addErrorMessage} from 'sentry/actionCreators/indicator';
+import type {MenuItemProps} from 'sentry/components/dropdownMenu';
+import {DropdownMenu} from 'sentry/components/dropdownMenu';
+import {IconDelete, IconDownload, IconEllipsis, IconUpload} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {defined} from 'sentry/utils';
+import {downloadObjectAsJson} from 'sentry/utils/downloadObjectAsJson';
+import {isActiveSuperuser} from 'sentry/utils/isActiveSuperuser';
+import useDeleteReplay from 'sentry/utils/replays/hooks/useDeleteReplay';
+import useShareReplayAtTimestamp from 'sentry/utils/replays/hooks/useShareReplayAtTimestamp';
+import type ReplayReader from 'sentry/utils/replays/replayReader';
+import {useIsSentryEmployee} from 'sentry/utils/useIsSentryEmployee';
+import {useNavigate} from 'sentry/utils/useNavigate';
+import useOrganization from 'sentry/utils/useOrganization';
+import type {ReplayRecord} from 'sentry/views/replays/types';
+
+interface Props {
+  projectSlug: string | null;
+  replay: ReplayReader | undefined;
+  // Accept the replay and replayRecord in case the replay doesn't load properly,
+  // we still want to be able to sent the Delete request.
+  replayRecord: ReplayRecord | undefined;
+}
+
+export default function ReplayItemDropdown({projectSlug, replay, replayRecord}: Props) {
+  const navigate = useNavigate();
+  const organization = useOrganization();
+  const isEmployee = useIsSentryEmployee();
+  const isSuperUser = isActiveSuperuser();
+
+  const replayId = replayRecord?.id;
+  const isMobile = replay?.isVideoReplay();
+
+  const canSeeEmployeeLinks = isEmployee || isSuperUser;
+  const canDownload = projectSlug && replay;
+
+  const onShareReplay = useShareReplayAtTimestamp();
+
+  const canDelete = replayId && projectSlug;
+  const onDeleteReplay = useDeleteReplay({replayId, projectSlug});
+
+  const dropdownItems: MenuItemProps[] = [
+    {
+      key: 'download-rrweb',
+      label: (
+        <ItemSpacer>
+          <IconDownload />
+          {t('Download JSON')}
+        </ItemSpacer>
+      ),
+      onAction: () => {
+        try {
+          if (!replay) {
+            addErrorMessage(t('Replay not found'));
+            return;
+          }
+          downloadObjectAsJson(replay.getRRWebFrames(), 'rrweb');
+        } catch (error) {
+          addErrorMessage(
+            'Could not export replay as rrweb data. Please wait or try again'
+          );
+        }
+      },
+      disabled: !canDownload,
+    },
+    canSeeEmployeeLinks && isMobile
```

Comment:
We're missing an employee-only item for the replay event

Reply:
added another link for the `replayRecord`